py-polars/polars/lazy/__init__.py

```diff
@@ -1,29 +1,39 @@
-from typing import Union, List, Callable, Optional, Dict
-
-from polars import Series
-from polars.frame import DataFrame, wrap_df
-from polars import datatypes
-from polars.datatypes import DataType
 import os
-import tempfile
-import subprocess
 import shutil
+import subprocess
+import tempfile
 from datetime import datetime
+from typing import (
+    Callable,
+    Dict,
+    List,
+    Optional,
+    Union,
+)
+
+from polars import (
+    Series,
```

Comment:
Hmm, seems that CI doesn't like this import now:
```
 Traceback:
/opt/hostedtoolcache/Python/3.6.13/x64/lib/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_db_benchmark.py:3: in <module>
    import polars as pl
/opt/hostedtoolcache/Python/3.6.13/x64/lib/python3.6/site-packages/polars/__init__.py:12: in <module>
    from .functions import *
/opt/hostedtoolcache/Python/3.6.13/x64/lib/python3.6/site-packages/polars/functions.py:22: in <module>
    from .lazy import LazyFrame
/opt/hostedtoolcache/Python/3.6.13/x64/lib/python3.6/site-packages/polars/lazy/__init__.py:14: in <module>
    from polars import (
E   ImportError: cannot import name 'Series'
```
Looks like this might be due to a circular import between the `__init__` files of `polars` and `lazy`. If so is that something we should fix?